CMakeLists.txt

```
@@ -36,22 +36,28 @@ option(GPU_DEVICE
 if("${GPU_DEVICE}" STREQUAL "OFF")
   set(GPU_DEVICE NONE)
 endif()
+
 if("${GPU_DEVICE}" STREQUAL "NONE")
+  message(STATUS "CPU-only build")
   set(GPU_DEVICE_CODE ${GPU_DEVICE_NONE})
 elseif("${GPU_DEVICE}" STREQUAL "CUDA")
+  message(STATUS "Building with CUDA support")
   set(GPU_DEVICE_CODE ${GPU_DEVICE_CUDA})
 elseif("${GPU_DEVICE}" STREQUAL "HIP")
   # As stated in the PyTorch documentation
-  # (https://docs.pytorch.org/docs/stable/notes/hip.html)
-  # > "PyTorch for HIP intentionally reuses the existing torch.cuda interfaces.
-  # > This helps to accelerate the porting of existing PyTorch code and models
-  # > because very few code changes are necessary, if any."
-  # Therefore we use the CUDA backend when HIP is selected. This is technically
-  # equivalent to specifying -DGPU_DEVICE=CUDA.
+  # (https://docs.pytorch.org/docs/stable/notes/hip.html) > "PyTorch for HIP
+  # intentionally reuses the existing torch.cuda interfaces. > This helps to
+  # accelerate the porting of existing PyTorch code and models > because very
+  # few code changes are necessary, if any." Therefore we use the CUDA backend
+  # when HIP is selected. This is technically equivalent to specifying
+  # -DGPU_DEVICE=CUDA.
```

Something funky happened to the formatting here:

```suggestion
  # (https://docs.pytorch.org/docs/stable/notes/hip.html) > "PyTorch for HIP
  # intentionally reuses the existing torch.cuda interfaces.
  # > This helps to accelerate the porting of existing PyTorch code and models
  # > because very few code changes are necessary, if any." 
  # Therefore we use the CUDA backend when HIP is selected. This is technically
  # equivalent to specifying -DGPU_DEVICE=CUDA.
```